benchmarks/railsbench/benchmark.rb

```diff
@@ -1,5 +1,17 @@
 require 'harness'
 
+# Before we activate Bundler, make sure gems are installed.
+# And before we load ActiveRecord, let's make sure the
+# database exists and is up to date.
+# Note: db:migrate will create the DB if it doesn't exist,
+# and this app's db/seeds.rb will delete and repopulate
+# the database, so rows shouldn't accumulate.
+Dir.chdir(__dir__) do
+  unless system({ 'RAILS_ENV' => 'production' }, "bundle install && bin/rails db:migrate db:seed")
```

Comment:
Is the `db:create` command from above unnecessary?

Reply:
In recent Rails versions if you run "bin/rails db:migrate" it will automatically create the database, yes. That wasn't true in earlier versions of Rails. I specifically tested with our railsbench.

I *am* having a bit of trouble with the "bundle install" half of this, though, on my AWS instance. So I'm going to mark this as draft and ask you not to merge it until I can track that part down.